pkg/storaged/client.js

```diff
@@ -602,6 +603,24 @@ client.older_than = function older_than(version) {
     return utils.compare_versions(this.manager.Version, version) < 0;
 };
 
+/* Mount users
+ */
+
+client.find_mount_users = (target, is_mounted) => {
+    if (is_mounted === undefined || is_mounted)
+        return python.spawn(mount_users_py, ["users", target], { superuser: "try", err: "message" }).then(JSON.parse);
+    else
+        return Promise.resolve([]);
+};
+
+client.stop_mount_users = (users) => {
+    if (users && users.length > 0) {
+        return python.spawn(mount_users_py, ["stop", JSON.stringify(users)],
+                            { superuser: "try", err: "message" });
+    } else
+        return Promise.resolve();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17670-20220825-201517-e2bf9819-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)